CoverflowAltTab@dmo60.de/manager.js

```diff
@@ -75,9 +75,9 @@ Manager.prototype = {
         win.delete(global.get_current_time());
     },
 
-    _startWindowSwitcher: function(display, screen, window, binding) {
+    _startWindowSwitcher: function(display, window, binding) {
```

Comment:
Oh My God. They actually _REMOVED_ a parameter on the callback function.... I thought gnome shell was done with breaking changes. Or maybe we are using an undocumented way of adding callbacks and there is a "good" one....who knows... I believe this extension needs a rewrite, but I don't have neither the time nor the skills required.